collectors/persistentvolumeclaim.go

```diff
@@ -26,15 +26,34 @@ import (
 )
 
 var (
+	descPersistentVolumeClaimInfo = prometheus.NewDesc(
+		"kube_persistentvolumeclaim_info",
+		"Information about persistent volume claim.",
+		[]string{
+			"namespace",
+			"storageclass",
+			"persistentvolumeclaim",
+		}, nil,
+	)
 	descPersistentVolumeClaimStatusPhase = prometheus.NewDesc(
 		"kube_persistentvolumeclaim_status_phase",
-		"The phase the claim is currently in.",
+		"The phase the persistent volume claim is currently in.",
 		[]string{
 			"namespace",
+			"storageclass",
```

Comment:
Understood now, thanks for explaining!

PR is [updated](https://github.com/kubernetes/kube-state-metrics/pull/226/files).